interactive/notifications.py

```diff
@@ -9,15 +11,19 @@
 env = Env()
 
 
-def notify_analysis_request_submitted(title, codelist, created_by):
+def notify_analysis_request_submitted(analysis_request, created_by):
```

Comment:
Is `created_by` a column on `analysis_request`?

Reply:
It is now, yes. Good point.